auth/src/main/java/org/aerogear/mobile/auth/AuthService.java

```diff
@@ -106,6 +109,7 @@ public void init(final Context context, final AuthServiceConfiguration authServi
         this.authStateManager = AuthStateManager.getInstance(context);
         this.authServiceConfiguration = authServiceConfiguration;
         this.oidcAuthenticatorImpl = new OIDCAuthenticatorImpl(this.serviceConfiguration, this.authServiceConfiguration, this.appContext, this.authStateManager);
+        MetricsRegistry.instance().registerProvider(new AuthMetricsProvider());
```

Comment:
I am not sure if it's just me, but `registerProvider` makes me think it is to register a provider for metrics service.  But it is really a provider to provide metrics data. So will it better to make it more clear?

Reply:
On IOS we call this `MetricsCollector`

Reply:
Good point, will also rename it to `MetricsCollector`. Provider in the sense: it provider metrics FROM this module TO the metrics service.